armi/nucDirectory/elements.py

```diff
@@ -25,124 +27,155 @@
-
         symbol : str
             element symbol
-
         name: str
             element name
-
+        phase : str
+            Chemical phase of the element at standard temperature and pressure (e.g., gas, liquid, solid).
+        group : str
+            Chemical group of the element.
         """
         self.z = z
         self.symbol = symbol
         self.name = name
+        self.phase = ChemicalPhase[phase]
+        self.group = ChemicalGroup[group]
         self.standardWeight = None
-        self.nuclideBases = []
-
-        other = byZ.get(z, None)
-        if other is not None and other == self:
-            raise Exception(
-                "Element with atomic weight {} already exists" "".format(self)
-            )
-        byZ[z] = self
-        byName[name] = self
-        bySymbol[symbol] = self
+        self.nuclides = []
+        addGlobalElement(self)
 
     def __repr__(self):
-        return "<Element {} {}>".format(self.symbol, self.z)
+        return f"<Element {self.symbol:>3s} (Z={self.z}), {self.name}, {self.group}, {self.phase}>"
 
     def __eq__(self, other):
         return (
             self.z == other.z
             and self.symbol == other.symbol
             and self.name == other.name
+            and self.phase == other.phase
+            and self.group == other.group
+            and len(self.nuclides) == len(other.nuclides)
         )
 
     def __hash__(self):
         return hash(self.name)
 
     def __iter__(self):
-        for nuc in self.nuclideBases:
+        for nuc in sorted(self.nuclides):
             yield nuc
 
     def append(self, nuclide):
-        self.nuclideBases.append(nuclide)
+        """Assigns and sorts the nuclide to the element and ensures no duplicates."""
+        from armi.nucDirectory.nuclideBases import NuclideBase
+
+        if nuclide in self.nuclides or not isinstance(nuclide, NuclideBase):
+            return
+        self.nuclides.append(nuclide)
+        self.nuclides = sorted(self.nuclides)
 
     def isNaturallyOccurring(self):
-        r"""
-        Calculates the total natural abundance and if this value is zero returns False.
-        If any isotopes are naturally occurring the total abundance will be >0 so it will return True
-        """
-        totalAbundance = 0.0
-        for nuc in self.nuclideBases:
-            totalAbundance += nuc.abundance
-        return totalAbundance > 0.0
+        """Return True if the element is occurs in nature."""
+        return any([nuc.abundance > 0.0 for nuc in self.nuclides])
 
     def getNaturalIsotopics(self):
+        """Return a list of nuclides that are naturally occurring for this element."""
+        return [nuc for nuc in self.nuclides if nuc.abundance > 0.0 and nuc.a > 0]
+
+    def isHeavyMetal(self):
         """
-        Return the nuclide bases of any naturally-occurring isotopes of this element.
+        Return True if the atomic number of greater than 89 (i.e., Z > 89).
```

Comment:
Perhaps you don't want to specifically cite "89" in the docstring, to avoid it becoming out of sync with `HEAVY_METAL_CUTOFF_Z`.